policyengine_us/variables/gov/states/dc/dhs/tanf/eligibility/dc_tanf_eligible.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class dc_tanf_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    label = "DC TANF eligible"
+    definition_period = YEAR
+    defined_for = StateCode.DC
+
+    def formula(spm_unit, period, parameters):
+        demographic_eligible = spm_unit("is_demographic_tanf_eligible", period)
+        income_eligible = spm_unit("dc_tanf_income_eligible", period)
+        resources_eligible = spm_unit("dc_tanf_resources_eligible", period)
+        return demographic_eligible and income_eligible and resources_eligible
```

Comment:
Needed for vectorization
```suggestion
        return demographic_eligible & income_eligible & resources_eligible
```

Reply:
If I use '&' instead of 'and', things will go wrong when I make test.

Reply:
What's the error?

Reply:
I could not make test now. I got this.
`coverage run -a --branch -m policyengine_core.scripts.policyengine_command test policyengine_us/tests/policy/ -c policyengine_us
process_begin: CreateProcess(NULL, coverage run -a --branch -m policyengine_core.scripts.policyengine_command test policyengine_us/tests/policy/ -c policyengine_us, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:9: test] Error 2`

Reply:
If I change 'and' back to '&'. I got this.
`policyengine_us/variables/gov/states/dc/dhs/tanf/eligibility/dc_tanf_eligible.py:15: in formula
    return demographic_eligible & income_eligible & resources_eligible
E   TypeError: ufunc 'bitwise_and' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''`
I searched on Internet and they said it is due to operator precedence. Maybe I try `return (demographic_eligible) & (income_eligible) & (resources_eligible)`

Reply:
I figure it out! I set float value type instead of bool for resouces_eligible. That's why it got error.